ansible/group_vars/vm_host/main.yml

```diff
@@ -9,7 +9,7 @@ skip_image_downloading: false
 
 vm_console_base: 7000
 memory: 2097152
-max_fp_num: 4
+max_fp_num: 6
```

Comment:
Would this change affect the current behavior of adding front panel ports to cEOS neighbor VMs?
The current behavior is to add 4 front panel ports to each cEOS VM. With this change, probably 6 front panel ports would be added.
Is there a method to avoid affecting the existing behavior of adding only 4 front panel ports?

Reply:
cEOS has more than 6 fp ports. This just provides the numbers of fp ports exporting from VM. Current sanity works, a.k.a just two more ports are exported from VM to the host. But no impact to existing behaviors.

Reply:
cEOS has 6 ports, but only 4 of them are front panel ports, 1 of them is management port, 1 of them is backplane port.

With this change, cEOS would have 8 ports, 6 of them would be front panel ports. 
Please refer to this code: https://github.com/sonic-net/sonic-mgmt/blob/master/ansible/roles/vm_set/library/ceos_network.py#L86 

This change breaks the existing behavior. Is there a way to avoid breaking the existing behavior? Like define another variable for using vsonic cisco neighbor VM?

Reply:
Fixed. Revert back default as 4 and use topology file to provide needed max_fp_ports.

https://github.com/sonic-net/sonic-mgmt/pull/13785/commits/9311513469cdc42b291723cda57be5a61b90c993

- name: Pick provided max num of fp
  set_fact:
    max_fp_num: "{{ max_fp_num_provided }}"
  when: max_fp_num_provided is defined

max_fp_num_provided : 6

Reply:
Is the "Revert back default as 4" change committed?

Reply:
Yes. I missed cherry-pick to PR branch.

The cherrypick commit is https://github.com/eddieruan-alibaba/sonic-mgmt/commit/9311513469cdc42b291723cda57be5a61b90c993

The final version is 

https://github.com/eddieruan-alibaba/sonic-mgmt/blob/eruan-testcasemaster/ansible/group_vars/vm_host/main.yml#L6